Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCStatsCollector.cpp

```diff
@@ -258,9 +265,8 @@ static inline void fillOutboundRtpStreamStats(RTCStatsReport::OutboundRtpStreamS
         stats.retransmittedPacketsSent = *rtcStats.retransmitted_packets_sent;
     if (rtcStats.retransmitted_bytes_sent.is_defined())
         stats.retransmittedBytesSent = *rtcStats.retransmitted_bytes_sent;
-    // Not Implemented
-    // if (rtcStats.ssrc.is_defined())
-    //     stats.rtxSsrc = *rtcStats.ssrc;
+    if (rtcStats.ssrc.is_defined())
+        stats.rtxSsrc = *rtcStats.ssrc;
```

Comment:
Should be modifying "stats.ssrc" instead of "stats.rtxSsrc" here

Reply:
This one is simply not needed, rtxSsrc is set below while ssrc is already set by another method.

Reply:
You're right 🫣